src/mixpanel-core.js

```diff
@@ -195,7 +195,7 @@ var create_mplib = function(token, config, name) {
  *     mixpanel.library_name.track(...);
  *
  * @param {String} token   Your Mixpanel API token
- * @param {Object} [config]  A dictionary of config options to override. <a href="https://github.com/mixpanel/mixpanel-js/blob/8b2e1f7b/src/mixpanel-core.js#L87-L110">See a list of default config options</a>.
+ * @param {Object} [config]  A dictionary of config options to override. <a href="https://github.com/mixpanel/mixpanel-js/blob/master/src/mixpanel-core.js#L86-L127">See a list of default config options</a>.
```

Comment:
This needs to be pinned to a particular commit ref (SHA), otherwise the line numbers will become incorrect when master changes.

Reply:
done. 👍